.github/workflows/ci.yml

```diff
@@ -120,22 +135,34 @@ jobs:
         run: |
           echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
 
+      - name: Pull snuba CI images
+        run: |
+          docker pull ${SNUBA_IMAGE}:${{ steps.branch.outputs.branch }} || \
+            docker pull ${SNUBA_IMAGE}:latest || true
+
       - name: Create Image
         run: |
           docker buildx create --name arm64-builder --use
-          docker buildx inspect --bootstrap
-          docker buildx build --platform linux/arm64 --cache-from $SNUBA_IMAGE:latest \
-            -t $SNUBA_IMAGE:${{ github.sha }} -t $SNUBA_IMAGE:latest --load .
+          docker buildx build --platform linux/arm64 \
+            --build-arg BUILDKIT_INLINE_CACHE=1 \
+            --cache-from ${SNUBA_IMAGE}:latest \
+            --cache-from ${SNUBA_IMAGE}:${{ steps.branch.outputs.branch }} \
+            -t ${SNUBA_IMAGE}:latest \
+            -t ${SNUBA_IMAGE}:${{ steps.branch.outputs.branch }} \
+            -t ${SNUBA_IMAGE}:${{ github.sha }} \
+            --load .
```

Comment:
This change is moving each argument into a new line, adding `--build-arg BUILDKIT_INLINE_CACHE=1` and using `--cache-from` for a branch.